openstack/loadbalancer/v2/pools/testing/requests_test.go

```diff
@@ -260,3 +260,68 @@ func TestUpdateMember(t *testing.T) {
 
 	th.CheckDeepEquals(t, MemberUpdated, *actual)
 }
+
+func TestUpdateMembers(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+	HandleMembersUpdateSuccessfully(t)
+
+	res := pools.UpdateMembers(fake.ServiceClient(), "332abe93-f488-41ba-870b-2ac66be7f853", pools.UpdateMembersOpts{
+		Members: []pools.CreateMemberOpts{
+			{
+				Name:         "web-server-1",
+				Weight:       20,
+				SubnetID:     "bbb35f84-35cc-4b2f-84c2-a6a29bba68aa",
+				Address:      "192.0.2.16",
+				ProtocolPort: 80,
+			},
+			{
+				Name:         "web-server-2",
+				Weight:       10,
+				SubnetID:     "bbb35f84-35cc-4b2f-84c2-a6a29bba68aa",
+				Address:      "192.0.2.17",
+				ProtocolPort: 80,
+			},
+		},
+	})
+	th.AssertNoErr(t, res.Err)
+}
+
+func TestRequiredUpdateMembersOpts(t *testing.T) {
```

Comment:
Thanks for all your suggestions, @jtopjian 

> BuildRequestBody is only being triggered on the UpdateMembersOpts struct and not each child CreateMemberOpts struct.
I am surprised because the current implementation works fine in my testing environment.

Reply:
Ooh, maybe I got you. The current implementation works but can not check the required field as expected, right?

Reply:
Yes, exactly :)